packages/@okta/i18n/src/properties/login_nl_NL.properties

```diff
@@ -559,7 +561,7 @@ password.complexity.no_last_name = Bevat niet uw achternaam
 # Requirements as HTML list.
 # These are copy of existing strings, but we need them since in a list, the starting letter has to be in CAPS and no period at the end.
 password.complexity.requirements.header = Wachtwoordvereisten:
-password.complexity.length.description = {0} tekens
+password.complexity.length.description = Ten minste {0} tekens
```

Comment:
@mauriciocastillosilva-okta  This is the Dutch mistranslation fix for https://oktainc.atlassian.net/browse/OKTA-362310.